packages/replicache/src/replicache-options.ts

```diff
@@ -264,6 +267,8 @@ export type ZeroReadOptions = {
  * Prevents us from creating any direct dependencies on Zero.
  */
 export interface ZeroOption {
+  set auth(auth: string);
```

Comment:
maybe this should just be `auth: string` in the type. Using a setter seems like an implementation detail